docker/compose.yaml

```diff
@@ -9,6 +9,7 @@ x-environment: &environment
   SUPERBLOCKS_AGENT_INTERNAL_HOST_AUTO: ${SUPERBLOCKS_AGENT_INTERNAL_HOST_AUTO:-true}
   SUPERBLOCKS_AGENT_PORT: ${SUPERBLOCKS_AGENT_PORT:-8020}
   SUPERBLOCKS_WORKER_METRICS_PORT: ${SUPERBLOCKS_WORKER_METRICS_PORT:-9090}
+  NODE_OPTIONS: --max_old_space_size=${SUPERBLOCKS_AGENT_MEMORY_LIMIT:-3000}
```

Comment:
Out of curiosity - do we know what's the average ram use of the agent under normal operations?

Reply:
We're looking at around 1Gi for the baseline usage. This could potentially spike depending on your workloads though.